src/HttpKernel/InstallationKernel.php

```diff
@@ -67,7 +67,6 @@ public function canBootRealSystem()
     {
         return file_exists($this->getRootDir().'/config/parameters.yml')
             && file_exists($this->getRootDir().'/../system/config/localconfig.php')
```

Comment:
What do you mean by *regular installation*? 😉 
Also, the check is running to see if Symfony can be booted. And due to the pull request that should work perfectly fine without localconfig, right? Just would need to be tested…